src/main/java/com/r307/arbitrader/service/model/ExchangeFee.java

```diff
@@ -0,0 +1,38 @@
+package com.r307.arbitrader.service.model;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+public class ExchangeFee {
+    private BigDecimal shortFee;
+    private BigDecimal longFee;
```

Comment:
Yeah I tried to be backwards compatible while at the same time explicit what is what and follow the same naming logic we have in the code where we have named things like `tradeFee` and `marginFee` for consistency reasons because we tend to use `shortSomething` and `longSomthing` in other parts of the code like `shortExchange`, `longExchange`, `shortLimitOrder`, `longLimitOrder`, `longTrade`, `shortTrade`, `longOpenOrders`, `shortOpenOrders` and so on.

So I think in the code `shortFee` and `longFee` keep the consistency and follow the naming we have been using,

But then on the `.xml` I am not sure how to name them because we have `margin` and `fee`. 

What do you think?

Reply:
I don't think they fit neatly into the "long" and "short" buckets like the other things do. The `longFee` isn't really that, it's also used for short trades in addition to the `marginFee`. It's a fee applied to **every** trade. The `marginFee` is only applied to margin trades and only when they open. Those happen to all be "short" so far in our model but I believe you'd still pay the `marginFee` if you opened a _long_ margin trade on Kraken?

Reply:
I renamed them to `tradeFee` and `marginFee` and also renamed on the `.yml` file

Reply:
Probably related to the 'rrr' thing but I don't see this fixed in this PR. There must be some changes that didn't get pushed correctly?